contracts/test/gmp/DestinationChainSwapExecutable.sol

```diff
@@ -22,13 +24,20 @@ contract DestinationChainSwapExecutable is AxelarExecutable {
     ) internal override {
         (string memory tokenSymbolB, string memory recipient) = abi.decode(payload, (string, string));
 
-        address tokenA = gateway.tokenAddresses(tokenSymbolA);
-        address tokenB = gateway.tokenAddresses(tokenSymbolB);
+        address tokenA = IAxelarGMPGatewayWithToken(address(gatewayAddress)).tokenAddresses(tokenSymbolA);
```

Comment:
```suggestion
        address tokenA = gatewayWithToken().tokenAddresses(tokenSymbolA);
```

Reply:
yeah, this way, it's more cleaner and I don't need to import the interface. updated.